layout.lua

```diff
@@ -7,38 +7,53 @@ package.path = script_dir .. "?.lua;" .. package.path
 local conkyrc = require('conkyrc')
 local polycore = require('src/polycore')
 local data = require('src/data')
-local widget = require('src/widget')
+local core  = require('src/widgets/core')
+local cpu   = require('src/widgets/cpu')
+local drive = require('src/widgets/drive')
+local gpu   = require('src/widgets/gpu')
+local mem   = require('src/widgets/memory')
+local net   = require('src/widgets/network')
+local text  = require('src/widgets/text')
```

Comment:
Please avoid aligning like this, it's tedious to maintain and makes diffs longer.

Reply:
Can do, I noticed you used aliases in some places but not others, if you'd prefer without I can make that change though.